v-next/hardhat-ignition/src/index.ts

```diff
@@ -754,3 +754,66 @@
 //     throw e;
 //   }
 // }
+
+import type { HardhatPlugin } from "@ignored/hardhat-vnext/types/plugins";
+
+import { emptyTask, task } from "@ignored/hardhat-vnext/config";
+import { ArgumentType } from "@ignored/hardhat-vnext/types/arguments";
+
+import { PLUGIN_ID } from "./internal/constants.js";
+
+const hardhatIgnitionPlugin: HardhatPlugin = {
+  id: PLUGIN_ID,
+  npmPackage: "@ignored/hardhat-vnext-ignition",
+  tasks: [
+    emptyTask("ignition", "Store your keys in a secure way").build(),
```

Comment:
Incorrect message

Reply:
I have updated the message to `"Deploy your smart contracts using Hardhat Ignition"`, taken directly from the Ignition scope entry in HH2.